cmd/helm/template.go

```diff
@@ -106,11 +106,17 @@ func newTemplateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 							if client.UseReleaseName {
 								newDir = filepath.Join(client.OutputDir, client.ReleaseName)
 							}
+							// fix https://github.com/helm/helm/issues/11481
+							// more info https://github.com/helm/helm/blob/7ed9d16dc764a5b94b378a7e217865efaa0d9ac8/cmd/helm/template.go#L200-L204
+							_, err := os.Stat(strings.Join([]string{newDir, m.Path}, string(filepath.Separator)))
```

Comment:
Why is `strings.Join` being used instead of `filepath.Join`? This second form is used just a few lines above. It has some filepath checking and handles the correct Separator.